src/containers/sign-on/utils/formatSocialProfile.ts

```diff
@@ -83,6 +88,13 @@ export const formatInstagramProfile = async (
       )
     }
   }
+  if ((instagramProfile.full_name?.length || 0) > MAX_DISPLAY_NAME_LENGTH) {
```

Comment:
in the case we have no full_name, wouldn't we want the user to review? (not sure if this happens in practice). i think the check maybe should be

`if (!instagramProfile.full_name || instagramProfile.full_name.length > MAX) {`